spa/cypress/e2e/01-donationPage.cy.js

```diff
@@ -806,6 +807,66 @@ describe('User flow: unhappy paths', () => {
   });
 });
 
+describe('StripePaymentForm unhappy paths', () => {
+  beforeEach(() => {
+    // We intercept requests for Google Recaptcha because in test env, sometimes the live recaptcha returns an error
+    // (possibly because we load it too many times successively???), which was causing test failure else where.
+    cy.intercept({ method: 'GET', url: 'https://www.google.com/recaptcha/*' }, { statusCode: 200 });
+    cy.interceptStripeApi();
+    cy.intercept(
+      { method: 'POST', url: getEndpoint(AUTHORIZE_STRIPE_PAYMENT_ROUTE) },
+      {
+        body: { provider_client_secret_id: fakeStripeSecret, email_hash: fakeEmailHash },
+        statusCode: 201
+      }
+    ).as('create-one-time-payment');
+    cy.visitDonationPage();
+    cy.get('[data-testid*="amount-120"]').click();
+    cy.get('[data-testid*="frequency-one_time"]').click();
+    fillOutDonorInfoSection();
+    fillOutAddressSection();
+    fillOutReasonForGiving();
+    cy.get('form')
+      .findByRole('button', { name: /Continue to Payment/ })
```

Comment:
```suggestion
      .findByRole('button', { name: 'Continue to Payment' })
```

If we need a loose match on this, I'd vote for `exact: false` in the function call.